server/src/main/java/com/google/speech/tools/voxetta/data/PromptBuilder.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.speech.tools.voxetta.data;
+
+import com.google.appengine.api.datastore.Entity;
+
+
+/**
+ * Builder pattern for Prompts.
+ */
+public class PromptBuilder {
```

Comment:
![image](https://user-images.githubusercontent.com/2927735/86183250-9bb55300-baff-11ea-9202-dd6b8db07980.png)

Reply:
thank you matt